### DISCUSSION TITLE
PIconnect on Oracle Linux

### DISCUSSION THREAD

**Discussion by rohanguptha** [Hugovdberg/PIconnect#619] [Q&A] [answered]
Hello,

I installed PIconnect package in my Oracle Linux server. I'm trying to connect to my PI Server but I can only see TestingAF. 
`{'TestingAF': {'server': <PIconnect.AFSDK.AF.PISystem object at 0x7f5e2c793290>, 'databases': {'TestDatabase': <PIconnect.AFSDK.AF.AFDatabase object at 0x7f5e2c793350>}}}`

Could you please guide me how can I connect to my PI Server using Oracle Linux? I was able to connect to PI Server using my laptop. I assume it worked because my laptop asset have access to PI Server.

**Comment by Hugovdberg** [comment, marked as answer]
This is related to issue #537, in essence it requires you to install the PI AF SDK on the server for Python to be able to use it. As far as I know this is not possible at this moment due to the dependency of the SDK on the .NET Framework 4.5 which in turn is dependent on Windows as the underlying OS. When OSIsoft provides a version of the SDK that depends on .NET 5 or later it will likely be possible as .NET 5 is available on all platforms.
